src/wp-includes/block-editor.php

```diff
@@ -245,7 +245,6 @@ function get_default_block_editor_settings() {
 			'media_image',
 			'media_gallery',
 			'media_video',
-			'meta',
```

Comment:
why is this removed?

Reply:
I think that's because we don't have a block with the same functionality.

Reply:
Correct. I ended up removing it in https://core.trac.wordpress.org/ticket/53301, though.